CriteoPublisherSdk/Tests/IntegrationTests/CRInterstitialDelegateTests.m

```diff
@@ -916,8 +951,12 @@ - (void)testCacheHasAdButAdContentFetchFailed {
       .andReturn(@"test?safearea");
   dependencyProvider.displaySizeInjector = displaySizeInjector;
 
-  OCMStub([mockCriteo getBid:[self expectedCacheAdUnit]])
-      .andReturn([self bidWithDisplayURL:@"test"]);
+  OCMStub([mockCriteo getBid:self.expectedCacheAdUnit responseHandler:[OCMArg any]])
```

Comment:
This snippet is used many times. It may be nice to wrap it in a named method.

Reply:
Done. Side note, many of these tests are lightly factorised with many repetitions which would deserve some rework.